recipe/run_test.py

```diff
@@ -1,4 +1,9 @@
 import sys
+import os
+
+# Use OpenBLAS with 1 thread only as it seems to be using too many
+# on the CIs apparently.
+os.environ["OPENBLAS_NUM_THREADS"] = "1"
```

Comment:
Sets `OPENBLAS_NUM_THREADS` to `1`. This should stop the segmentation fault on Travis CI.